Kickstarter-iOS/DataSources/ProjectPamphletContentDataSource.swift

```diff
@@ -25,8 +25,8 @@ internal final class ProjectPamphletContentDataSource: ValueCellDataSource {
     let liveStreamSubpages = self.liveStreamSubpages(forLiveStreamEvents: liveStreamEvents)
 
     let values = liveStreamSubpages + [
-      .comments(project.stats.commentsCount ?? 0, liveStreamSubpages.isEmpty ? .first : .middle),
-      .updates(project.stats.updatesCount ?? 0, .last)
+      .comments(project.stats.commentsCount as Int?, liveStreamSubpages.isEmpty ? .first : .middle),
+      .updates(project.stats.updatesCount as Int?, .last)
```

Comment:
Initially had this here for doing animation in the subpage cell, but didn't need the animation after all. Decided to leave the coalescing to the view model because it felt better.